.travis.yml

```diff
@@ -50,18 +50,22 @@ script:
     if [[ "$BRANCH" == "master" ]];
     then
       if [[ "$PROJECT" != "common" ]]; then
-        sbt "project $PROJECT" -DconfigBucket=$CONFIG_BUCKET -Denv=$BUILD_ENV ecr:push;
+        sbt "project $PROJECT" stage;
+        export NAME="0.0.1-$(git rev-parse HEAD)_alex";
```

Comment:
This name is deliberate – at least initially, it will produce broken images, and I don’t want to deploy to prod until I’ve got the infra pieces in place as well.